cng/ecdh.go

```diff
@@ -126,22 +126,17 @@ func GenerateKeyECDH(curve string) (*PrivateKeyECDH, []byte, error) {
 	// GenerateKeyECDH returns the public key as as byte slice.
 	// To get it we need to export the raw CNG key blob
 	// and prepend the encoding prefix.
-	hdr, blob, err := exportECCKey(hkey, false)
+	hdr, bytes, err := exportECCKey(hkey, true)
```

Comment:
Hmm, noticing now that it might be a slightly better API design to make typed const names for public/private -> true/false to make the difference clearer at the call site. Although it's not super important here since it's not exported.

Reply:
If it was an exported function, I would agree with you, but I like the simplicity of booleans for internal things :)